### PR TITLE
refactor script to a file and show notification in browser

### PR DESCRIPTION
**Pull request by mauricionr**
Refactor and show notification

## Description

Moved the script in example to a file and show notification 

## Motivation and Context

it's better than console.log 🙈 

## How Has This Been Tested?

Access the browser page and accept the notification, then make the curl

followed the readme from the test and increment the code

## Screenshots (if appropriate):
screen shot 2019-01-30 at 00 13 14

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/test_frontend_socket/assets/app.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    };
+    var n = new Notification(title, options);
+}
+socket.connect()
+
+joinChannel("test")
+joinChannel("test2")
+
+pushex.subscribe('test').bind('*', (event, data) => {
+  spawnNotification('body', 'icon', 'title')
```
Comment (sb8244) [inline examples/test_frontend_socket/assets/app.js]: Do you intend for these to contain the data?

File: examples/test_frontend_socket/assets/app.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        if (permission === "granted") {
+          var notification = new Notification("Hi there!");
+        }
+      });
+    }
+  
+    // At last, if the user has denied notifications, and you 
+    // want to be respectful there is no need to bother them any more.
+  }
+Notification.requestPermission().then(function(result) {
```
Comment (sb8244) [inline examples/test_frontend_socket/assets/app.js]: I'll run this through formatting after merge to ensure consistency with rest of style

File: examples/test_frontend_socket/assets/app.js (hunk 1 above)
Comment (mauricionr) [reply examples/test_frontend_socket/assets/app.js]: Yes, i already did. 🍻

**Comment by mauricionr** [comment]
@sb8244 now i'm using the event and data, i also put the pushex icon.

**Comment by sb8244** [comment]
Thanks so much @mauricionr . You can see my changes in the merged branch that I made...not very many though!

**Pull request merged**